modules/phase_field/test/tests/boundary_intersecting_features/tests

```diff
@@ -5,5 +5,6 @@
     csvdiff = boundary_intersecting_features_out_grain_volumes_0000.csv
     # This test requires VTK because it uses the ImageFunction class
     vtk = true
+    max_parallel = 1 # See #9886
```

Comment:
This test doesn't use the NODAL mode. It does however contain a misleading AuxKernel with the word nodal in it. Did you by chance get a false positive here?

Reply:
Yeah, I did.  Reenabling now.

Reply:
False positive in the "trying to grep and read through test input files" sense, I mean.